### PR TITLE
Feat:Alert test

### PR DESCRIPTION
**Pull request by STEPHINRACHEL**

### CODE REVIEW

**Comment by jhellar** [comment]
@STEPHINRACHEL looks good to me, have just few comments:

- PR will probably need to be rebased on master
- change `OPENSHIFT_USER`/`OPENSHIFT_PASS` env vars to `OPENSHIFT_USERNAME`/`OPENSHIFT_PASSWORD` so that we can share same job definition [1] for all test suites

[1] https://github.com/RHCloudServices/centralci/blob/master/jobs-dsl/aerogear.groovy#L13-L14

File: backend/test/alerts.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        }, 7 * 60000);
+        await exec(`kubectl scale --replicas=1 deployments unifiedpush-operator -n ${pushNamespace}`);
+    })
+})
+const getParam = async() => {
+    // const ns = await exec("oc projects -q | grep middleware-monitoring");
+    // const monitoringNameSpace = ns.stdout.slice(0, ns.stdout.length - 1);
+    monitoringNameSpace = await getNameSpaces("middleware-monitoring");
+    const routes = await resource(TYPE.ROUTE, ACTION.GET_ALL, null, monitoringNameSpace);
+    alertUrl = `https://${routes.items[2].spec.host}/api/v1/alerts`;
```
Comment (jhellar) [inline backend/test/alerts.js]: Not sure if we can rely on this (route being 2nd every time in the list). Should be possible to find the correct route based on its name or something like that.

File: backend/test/alerts.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+describe('MDC alerts test', function() {
+    this.timeout(0);
+    before('authenticate with oauth proxy', async() => {
+        await init();
+        mdcNamespace = await getNameSpaces("mobile-developer-console");
+        await getParam();
+    });
+
+    it('should trigger mdc down alert', async() => {
+        await exec(`oc scale --replicas=0 dc mdc -n ${mdcNamespace}`);
```
Comment (b1zzu) [inline backend/test/alerts.js]: Hey could we stick to oc or kubectly, I don't know if there is a reason to use one over the other but I wouldn't use both
```
await exec(`kubectl scale --replicas=0 deployment/mobile-developer-console -n ${mdcNamespace}`)
```

Comment (STEPHINRACHEL) [reply backend/test/alerts.js]: yes we can

**Pull request closed without merging**